wg-node-lifecycle/README.md

```diff
@@ -0,0 +1,48 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Node Lifecycle Working Group
+
+Explore and improve node and pod lifecycle in Kubernetes. This should result in better node drain/maintenance support and better pod disruption/termination. It should also improve node and pod autoscaling, better application migration and availability, load balancing, de/scheduling, node shutdown, cloud provider integrations, and support other new scenarios and integrations.
```

Comment:
This statement is quite broad at the moment, I think we probably want to revise this to be more succinct once we have worked out our initial priorities

Reply:
It is broad, but it captures the impact of our 3 focus features, so I would be in favor of keeping the statement if we have the priorities well defined and exit criteria updated over time.